### PR TITLE
Minor fixes while building openjdk in native env (without docker)

### PR DESCRIPTION
**Pull request by neomatrix369**
Fixed a abs/relative path issue, also replaced hard-coded references to openjdk repo folder name with a variable

### CODE REVIEW

File: sbin/build.sh [hunk 1]
```diff
@@ -95,7 +96,7 @@ else
   fi
    
   # We get the files we need at $WORKING_DIR/installedfreetype
-  ./configure --prefix=$WORKING_DIR/installedfreetype $PARAMS && make all && make install
+  ./configure --prefix=$WORKING_DIR/openjdk/installedfreetype $PARAMS && make all && make install
```
Comment (karianna) [inline sbin/build.sh]: Shouldn't this now be `$OPENJDK_REPO_NAME` as well?

Comment (neomatrix369) [inline sbin/build.sh]: Yes, good catch!

**Comment by neomatrix369** [comment]
@karianna this is more a bug-fix rather than an enhancement, fixing a folder reference issue in `sbin/build.sh` (just for posterity reasons good to label it closer to what it means)

File: makejdk.sh [hunk 2]
```diff
@@ -119,25 +120,25 @@ else
 fi
 
 echo $git
-if [ -d "$WORKING_DIR"/openjdk/.git ] && [ $REPOSITORY == "AdoptOpenJDK/openjdk-jdk8u" ] ; then
+if [ -d "$WORKING_DIR"/$OPENJDK_REPO_NAME/.git ] && [ $REPOSITORY == "AdoptOpenJDK/openjdk-jdk8u" ] ; then
   # It does exist and it's a repo other than the AdoptOpenJDK one
```
Comment (karianna) [inline makejdk.sh]: I assume a later PR will take care of the hard coding of `openjdk-jdk8u`?

**Comment by karianna** [comment]
@gdams - If you're happy with this, then lets merge it :-)

File: makejdk.sh (hunk 2 above)
Comment (neomatrix369) [inline makejdk.sh]: Will create multiple PRs to move script to more generic and reusable state - focused PRs

**Pull request merged**